mayastor/Cargo.toml

```diff
@@ -85,10 +85,9 @@ udev = "0.6.2"
 url = "2.2.2"
 async-channel = "1.6.1"
 dns-lookup = "1.0.8"
-mbus_api = { path = "../mbus-api" }
 etcd-client = "0.7.1"
 parking_lot = "0.11.1"
-
+strum_macros = "0.21.1"
```

Comment:
What is this used for?

Reply:
` use strum_macros::{AsRefStr, ToString};` we are using these from this crate for the ReplyError type